providers/celery/src/airflow/providers/celery/cli/celery_command.py

```diff
@@ -135,7 +135,8 @@ def _run_stale_bundle_cleanup():
         try:
             yield
         finally:
-            return
+            pass  # ignore any problems
+        return
```

Comment:
This is tricky -- it causes all exceptions to be bubbled up, not ignored!

`finally-pass` is essentially a no-op.

Instead, I suggest:

```suggestion
        with suppress(BaseException):
            yield
        return
```

(`suppress` is already being used a few lines up)

Reply:
Wondering if these lines were covered.

Reply:
Thanks, scatched my head about this, your proposal is much better!

Reply:
Nice!